app/views/admin/analysis/generic_chart_template.html.erb

```diff
@@ -18,7 +18,9 @@ with <%= @school.number_of_pupils %> pupils and a floor area of <%= @school.floo
     <hr class="analysis"/>
   <% end %>
   <div id="chart_wrapper_<%= chart %>" class="chart-wrapper">
-    <h3 class="analysis">Loading: <%= chart.to_s.humanize %></h3>
+    <div class="dynamic-titles">
+      <h3 class="analysis">Loading: <%= chart.to_s.humanize %></h3>
```

Comment:
does "Loading" need extracting to a translation yml file?

Reply:
This is an admin only page, that will get retired soon.